packages/docs/site/docs/02-start-using/01-index.md

```diff
@@ -103,7 +103,7 @@ You can specify major versions like `wp=6.2` or `php=8.1` and expect the most re
 
 ## Import a WXR file
 
-You can import a WordPress export file by uploading a WXR file in [/wp-admin/](https://playground.wordpress.net/wp-admin/).
+You can import a WordPress export file by uploading a WXR file in [/wp-admin/](https://playground.wordpress.net/?url=/wp-admin/import.php).
```

Comment:
Technically it's a separate change but let's just keep it in this PR, separating it is not worth the effort

Reply:
Sounds good